modules/swagger-codegen/src/main/resources/typescript-fetch/api.mustache

```diff
@@ -76,7 +76,7 @@ export class RequiredError extends Error {
  * {{&description}}{{/description}}
  * @export
  */
-export const {{classname}}FetchParamCreator = function (configuration?: Configuration) {
+export const {{classname}}FetchParamCreator = function ({{#lambda.lowercase}}{{apiConfigClassName}}{{/lambda.lowercase}}?: {{#lambda.titlecase}}{{apiConfigClassName}}{{/lambda.titlecase}}) {
```

Comment:
why need to change the parameter name?

Reply:
I'll reply here on this even thought this specific line is getting ready to change when I make a commit here in a few minutes.  `lambda.lowercase` is going to change to `lambda.camelcase_param`.

I agree it does make the template a bit harder to read, but the parameter name is changing to this as it will now be dynamic in nature based on what you pass into the generator. 

That's the basic point of this pull request.  There are swagger definitions that may auto-generate other parameters/variables named `configuration` in the same class.  When this happens you end up with invalid code as you can't have two parameters/variables with the same name.

Reply:
@macjohnny, if you haven't yet, take a look at https://github.com/swagger-api/swagger-codegen/issues/7911.  I include a sample swagger definition where if it's run against the generator, you will end up with multiple parameters/variables named `configuration`.

I would also add that the reason I'm putting the `lambda.camelcase_param` is to try and conform to standard Javascript/Typescript patterns that has Class/Interface names as PascalCase and parameters/variables in camelCase.    By using the lambda's, I'm allowing someone to pass in the dynamic Configuration class name as something like `ApiConfiguration` or `apiConfiguration` and the template will then know when to use PascalCase or camelCase automatically based on those standards.